pandas_gbq/gbq.py

```diff
@@ -581,7 +581,11 @@ def load_data(self, dataframe, dataset_id, table_id, chunksize):
                 self._print("\rLoad is {0}% Complete".format(
                     ((total_rows - remaining_rows) * 100) / total_rows))
 
-                body = StringIO('{}\n'.format('\n'.join(rows)))
+                body = '{}\n'.format('\n'.join(rows))
```

Comment:
If you use `u'{}\n'.format()` is the if statement checking for `bytes` necessary?

Reply:
Unfortunately not:
```

>               body = u'{}\n'.format(u'\n'.join(rows))
E               UnicodeDecodeError: 'ascii' codec can't decode byte 0xe4 in position 77: ordinal not in range(128)
```

I _think_ the nub problem is that `row.to_json` comes out as either bytes or str depending on the python version - so we need some branching somewhere. Unless there's a function in python that can deal with both (this all seems a bit inelegant)

(I also tried decoding the row first on 576, which made Py2 pass, but then python3 failed, because it can't decode unicode.